### ISSUE TITLE
欢迎提交未收录的代码

### ISSUE THREAD

**Issue by yshenaw** [IIT-Lab/Paper-with-Code-of-Wireless-communication-Based-on-DL#15]
如果你知道一些相关的开源论文，但不在此列表中，非常欢迎添加在此issue当中，感谢为开源社区贡献一份力量

**Comment by recusant7** [comment]
Xia, J., He, K., Xu, W., Zhang, S., Fan, L., & Karagiannidis, G. K. (2020). A MIMO detector with deep learning in the presence of correlated interference. IEEE Transactions on Vehicular Technology, 69(4), 4492-4497
https://github.com/skypitcher/project_dcnnmld

**Comment by revenol** [comment]
L. Huang, S. Bi, and Y. J. Zhang, “Deep reinforcement learning for online computation offloading in wireless powered mobile-edge computing networks,” IEEE Trans. Mobile Compt., vol. 19, no. 11, pp. 2581-2593, November 2020.

https://github.com/revenol/DROO

**Comment by linkpark** [comment]
J. Wang, J.Hu, G. Min, A. Y. Zomaya, and N. Georgalas, “Fast Adaptive Computation Offloading in Edge Computing based on Meta Reinforcement Learning“. IEEE Transactions on Parallel and Distributed Systems, vol. 32, no. 1, pp. 242--253, 2021.

https://arxiv.org/abs/2008.02033

https://github.com/linkpark/metarl-offloading

**Comment by Azhe9510** [comment]
建议在表格的每一行前添加序号 方便大家知道最近添加的是哪些论文代码

**Comment by zhuwenxing** [comment]
> 建议在表格的每一行前添加序号 方便大家知道最近添加的是哪些论文代码

一般最前面的就是新添加的，但是不能保证新添加论文发表的时间也是最新的。

**Comment by STARainZ** [comment]
K. Pratik, B. D. Rao, and M. Welling, “RE-MIMO: Recurrent and Permutation Equivariant Neural MIMO Detection,” IEEE Trans. Signal Process., vol. 69, pp. 459–473, 2021.

https://arxiv.org/abs/2007.00140

https://github.com/krpratik/RE-MIMO

**Comment by Bella720** [comment]
F. B. Mismar, A. Alammouri, A. Alkhateeb, J. G. Andrews, and B. L. Evans, “Deep Learning Predictive Band Switching in Wireless Networks,” IEEE Transactions on Wireless Communications, vol. 20, no. 1, pp. 96–109, Jan. 2021, doi: 10.1109/TWC.2020.3023397.

https://ieeexplore.ieee.org/abstract/document/9199558

https://github.com/farismismar/Bandswitch-DeepMIMO

**Comment by tanxiangtj** [comment]
H. Chang, H. Song, Y. Yi, J. Zhang, H. He and L. Liu, "Distributive Dynamic Spectrum Access Through Deep Reinforcement Learning: A Reservoir Computing-Based Approach," in IEEE Internet of Things Journal, vol. 6, no. 2, pp. 1938-1948, April 2019, doi: 10.1109/JIOT.2018.2872441.

https://ieeexplore.ieee.org/document/8474348
https://github.com/haohsuan2918/DQN_RC_DSA_IOT2019

**Comment by tanxiangtj** [comment]
https://mlc.committees.comsoc.org/papers-with-code/

**Comment by revenol** [comment]
Suzhi Bi, Liang Huang, Hui Wang, and Ying-Jun Angela Zhang, "Lyapunov-guided Deep Reinforcement Learning for Stable Online Computation Offloading in Mobile-Edge Computing Networks," IEEE Transactions on Wireless Communications, 2021, doi:10.1109/TWC.2021.3085319.

https://ieeexplore.ieee.org/document/9449944
https://github.com/revenol/LyDROO

**Comment by STARainZ** [comment]
H. He, C. Wen, S. Jin, and G. Y. Li, “Model-driven deep learning for MIMO detection,” IEEE Trans. Signal Process., vol. 68, pp. 1702–1715, Feb. 2020.

https://ieeexplore.ieee.org/document/9018199/
https://github.com/hehengtao/OAMP-Net

**Comment by silentsaber** [comment]
J. Choi, Y. Cho, B. L. Evans and A. Gatherer, "Robust Learning-Based ML Detection for Massive MIMO Systems with One-Bit Quantized Signals," 2019 IEEE Global Communications Conference (GLOBECOM), 2019, pp. 1-6, doi: 10.1109/GLOBECOM38437.2019.9013332.

https://github.com/Yunseong-Cho/LearningML

**Comment by xiangli-bjtu** [comment]
Lee M, Yu G, Li G Y. Graph Embedding-Based Wireless Link Scheduling With Few Training Samples[J]. IEEE Transactions on Wireless Communications, 2020, 20(4): 2282-2294.

https://github.com/mengyuan-lee/graph_embedding_link_scheduling

**Comment by ruihuili** [comment]
Li, Rui, Ondrej Bohdal, Rajesh K. Mishra, Hyeji Kim, Da Li, Nicholas Donald Lane, and Timothy Hospedales. "A Channel Coding Benchmark for Meta-Learning." In Thirty-fifth Conference on Neural Information Processing Systems Datasets and Benchmarks Track. 2021.  

https://openreview.net/forum?id=DjzPaX8AT0z 
https://github.com/ruihuili/MetaCC

**Comment by linkpark** [comment]
J. Wang, J.Hu, G. Min, W. Zhan, A. Y. Zomaya, and N. Georgalas, "Dependent Task Offloading for Edge Computing based on Deep Reinforcement Learning." IEEE Transactions on Computers, 2021.

https://ieeexplore.ieee.org/abstract/document/9627763 
https://github.com/linkpark/RLTaskOffloading

**Comment by hqyyqh888** [comment]
Q. Hu, Y. Liu, Y. Cai, G. Yu, and Z. Ding, “Joint deep reinforcement learning and unfolding: Beam selection and precoding for
mmWave multiuser MIMO with lens arrays,” IEEE J. Sel. Areas Commun., vol. 39, no. 8, pp. 2289–2304, Jun. 2021.

https://ieeexplore.ieee.org/stamp/stamp.jsp?tp=&arnumber=9448095
https://github.com/hqyyqh888/DDQN_BeamSelection

**Comment by linkpark** [comment]
您好，您的来信我已收到。

**Comment by wsyCUHK** [comment]
S. Wang, S. Bi and Y. -J. A. Zhang, "Deep Reinforcement Learning with Communication Transformer for Adaptive Live Streaming in Wireless Edge Networks," in IEEE Journal on Selected Areas in Communications, doi: 10.1109/JSAC.2021.3126062.

https://ieeexplore.ieee.org/document/9605672
https://github.com/wsyCUHK/SACCT

**Comment by silentsaber** [comment]
H. Lu, M. Jiang and J. Cheng, "Deep Learning Aided Robust Joint Channel Classification, Channel Estimation, and Signal Detection for Underwater Optical Communication," in IEEE Transactions on Communications, vol. 69, no. 4, pp. 2290-2303, April 2021, doi: 10.1109/TCOMM.2020.3046659.

https://ieeexplore.ieee.org/document/9302692
https://github.com/Huaiyin-Lu/UWOC-JCCESD

**Comment by liyiq5** [comment]
邮件已经收到，谢谢，祝好~！

**Comment by Charlie258** [comment]
你好，你的邮件已经收到，谢谢。

**Comment by linkpark** [comment]
您好，您的来信我已收到。

**Comment by DrizzlingVoid** [comment]
Z. He, L. Wang, H. Ye, G. Y. Li and B. -H. F. Juang, "Resource Allocation based on Graph Neural Networks in Vehicular Communications," GLOBECOM 2020 - 2020 IEEE Global Communications Conference, 2020, pp. 1-5, doi: 10.1109/GLOBECOM42002.2020.9322537.

https://github.com/Coolzyh/Globecom2020-ResourceAllocationGNN

**Comment by liyiq5** [comment]
邮件已经收到，谢谢，祝好~！

**Comment by qingshuangyixia** [comment]
群人数超过200，进不去了。

**Comment by liyiq5** [comment]
邮件已经收到，谢谢，祝好~！

**Comment by Charlie258** [comment]
你好，你的邮件已经收到，谢谢。

**Comment by linkpark** [comment]
Wang, J., Hu, J., Min, G., Ni, Q., & El-Ghazawi, T. (2022). Online Service Migration in Mobile Edge with Incomplete System Information: A Deep Recurrent Actor-Critic Learning Approach. IEEE Transactions on Mobile Computing.

https://ieeexplore.ieee.org/document/9853218
https://github.com/linkpark/pomdp-service-migration

**Comment by liyiq5** [comment]
邮件已经收到，谢谢，祝好~！

**Comment by Charlie258** [comment]
你好，你的邮件已经收到，谢谢。

**Comment by linkpark** [comment]
您好，您的来信我已收到。

**Comment by Msixram** [comment]
L. Zhang, J. Tan, Y. -C. Liang, G. Feng and D. Niyato, "Deep Reinforcement Learning-Based Modulation and Coding Scheme Selection in Cognitive Heterogeneous Networks," in IEEE Transactions on Wireless Communications, vol. 18, no. 6, pp. 3281-3294, June 2019, doi: 10.1109/TWC.2019.2912754.

url:https://ieeexplore.ieee.org/document/8703432

**Comment by linkpark** [comment]
您好，您的来信我已收到。

**Comment by liyiq5** [comment]
邮件已经收到，谢谢，祝好~！

**Comment by Charlie258** [comment]
你好，你的邮件已经收到，谢谢。

**Comment by 18140200203-mwh** [comment]
F. Saggese, L. Pasqualini, M. Moretti and A. Abrardo, "Deep Reinforcement Learning for URLLC data management on top of scheduled eMBB traffic," 2021 IEEE Global Communications Conference (GLOBECOM), 2021, pp. 1-6, doi: 10.1109/GLOBECOM46510.2021.9685777.

L. Pasqualini and F. Saggese, "Deep Reinforcement Learning for URLLC data management on top of scheduled eMBB traffic", GitHub repository, 2021, [online] Available: https://github.com/InsaneMonster/telerl2021.

**Comment by linkpark** [comment]
您好，您的来信我已收到。

**Comment by liyiq5** [comment]
邮件已经收到，谢谢，祝好~！

**Comment by Charlie258** [comment]
你好，你的邮件已经收到，谢谢。

**Comment by 18140200203-mwh** [comment]
您的邮件我已经收到 感谢！

**Comment by zhongyuanzhao** [comment]
[Zhongyuan Zhao](https://openreview.net/profile?id=~Zhongyuan_Zhao1), [Ananthram Swami](https://openreview.net/profile?id=~Ananthram_Swami1), [Santiago Segarra](https://openreview.net/profile?id=~Santiago_Segarra1), Graph-based Deterministic Policy Gradient for Repetitive Combinatorial Optimization Problems, ICLR 2023 

https://openreview.net/pdf?id=yHIIM9BgOo
https://github.com/XzrTGMu/twin-nphard

这篇最新收录于ICLR2023的文章提出一种通用的强化学习架构，用以解决重复性的组合优化问题，尤其是适合需要快速、或者分布式方案的应用。文中演示了4种不同的NP-hard组合优化问题,MWIS, MWDS, NWST, MWCDS，这几个问题常见于通信网络的调度和路由等资源分配任务中。

**Comment by WeihaoMao** [comment]
您的邮件我已经收到 感谢！

**Comment by Charlie258** [comment]
你好，你的邮件已经收到，谢谢。

**Comment by linkpark** [comment]
您好，您的来信我已收到。

**Comment by liyiq5** [comment]
邮件已经收到，谢谢，祝好~！

**Comment by LearnUgly** [comment]
您好，您的邮件已经收到，谢谢！



在 2023-03-04 21:26:14，"Zhongyuan Zhao" ***@***.***> 写道：

Zhongyuan Zhao, Ananthram Swami, Santiago Segarra, Graph-based Deterministic Policy Gradient for Repetitive Combinatorial Optimization Problems, ICLR 2023

https://openreview.net/pdf?id=yHIIM9BgOo
https://github.com/XzrTGMu/twin-nphard

这篇最新收录于ICLR2023的文章提出一种通用的强化学习架构，用以解决重复性的组合优化问题，尤其是适合需要快速、或者分布式方案的应用。文中演示了4种不同的NP-hard组合优化问题,MWIS, MWDS, NWST, MWCDS，这几个问题常见于通信网络的调度和路由等资源分配任务中。

**Comment by wsyCUHK** [comment]
S. Wang, S. Bi and Y. -J. A. Zhang, "Edge Video Analytics with Adaptive Information Gathering: A Deep Reinforcement Learning Approach," in IEEE Transactions on Wireless Communications, doi: 10.1109/TWC.2023.3237202. https://ieeexplore.ieee.org/document/10025689
https://github.com/wsyCUHK/DBAG

**Comment by yikuanglancheng** [comment]
Z. Ren, N. Cheng, R. Sun, X. Wang, N. Lu and W. Xu, "SigT: An Efficient End-to-End MIMO-OFDM Receiver Framework Based on Transformer," 2022 5th International Conference on Communications, Signal Processing, and their Applications (ICCSPA), Cairo, Egypt, 2022, pp. 1-6, doi: 10.1109/ICCSPA55860.2022.10019001.

https://github.com/SigTransformer/SigT

**Comment by WeihaoMao** [comment]
您的邮件我已经收到 感谢！

**Comment by linkpark** [comment]
您好，您的来信我已收到。

**Comment by wsyCUHK** [comment]
S. Wang, J. Yang and S. Bi, "Adaptive Video Streaming in Multi-Tier Computing Networks: Joint Edge Transcoding and Client Enhancement," in IEEE Transactions on Mobile Computing, doi: 10.1109/TMC.2023.3263046. https://ieeexplore.ieee.org/document/10089145

https://github.com/Max-JY-Young/EDTEA

**Comment by gaze-wu** [comment]
paper ： https://arxiv.org/pdf/2305.07309.pdf

code ： https://github.com/tomerraviv95/facilitating-adaptation-deep-receivers

**Comment by WeihaoMao** [comment]
您的邮件我已经收到 感谢！

**Comment by Zmjcc** [comment]
paper: https://ieeexplore.ieee.org/abstract/document/9834153
code: https://github.com/MauroZMJ/Deep_Learning_based_Low_complexity_MIMO_Precoding_Design

**Comment by WeihaoMao** [comment]
您的邮件我已经收到 感谢！

**Comment by liyiq5** [comment]
邮件已经收到，谢谢，祝好~！

**Comment by Charlie258** [comment]
你好，你的邮件已经收到，谢谢。

**Comment by chang2727** [comment]
您好。最近我在研究利用强化学习解决双能源供电的通信网络中资源优化问题，但发现几乎没有近几年开源的代码。请问大家有知道的吗？谢谢！

**Comment by liyiq5** [comment]
邮件已经收到，谢谢，祝好~！

**Comment by linkpark** [comment]
您好，您的来信我已收到。

**Comment by Charlie258** [comment]
你好，你的邮件已经收到，谢谢。

**Comment by WeihaoMao** [comment]
您的邮件我已经收到 感谢！

**Comment by MohamedNennouche** [comment]
Paper : "Meta-ViterbiNet: Online Meta-Learned Viterbi Equalization for Non-Stationary Channels"
https://arxiv.org/abs/2103.13483

Code : https://github.com/tomerraviv95/meta-viterbinet?tab=readme-ov-file

**Comment by liyiq5** [comment]
邮件已经收到，谢谢，祝好~！

**Comment by Charlie258** [comment]
你好，你的邮件已经收到，谢谢。

**Comment by linkpark** [comment]
您好，您的来信我已收到。

**Comment by WeihaoMao** [comment]
您的邮件我已经收到 感谢！

**Comment by FenghaoZhu** [comment]
Paper : "Robust Beamforming for RIS-aided Communications: Gradient-based Manifold Meta Learning"
https://ieeexplore.ieee.org/document/10623434

Code : https://github.com/FenghaoZhu/GMML

**Comment by Charlie258** [comment]
你好，你的邮件已经收到，谢谢。

**Comment by WeihaoMao** [comment]
您的邮件我已经收到 感谢！

**Comment by xingtianqz** [comment]
您好，邮件已收到，谢谢。赵晓蕾

**Comment by linkpark** [comment]
您好，您的来信我已收到。

**Comment by liyiq5** [comment]
邮件已经收到，谢谢，祝好~！

**Comment by FenghaoZhu** [comment]
Paper : "Robust Beamforming with Gradient-based Liquid Neural Network"
https://ieeexplore.ieee.org/document/10620247

Code : https://github.com/tp1000d/GLNN

**Comment by zhuwenxing** [comment]
> Paper : "Robust Beamforming with Gradient-based Liquid Neural Network" https://ieeexplore.ieee.org/document/10620247
> 
> Code : https://github.com/tp1000d/GLNN

@FenghaoZhu 
Thank you for your submission.
Already added in https://github.com/zhuwenxing/daily_arxiv?tab=readme-ov-file#2024-08-05
If you want to add to this repo, can you submit a PR?

**Comment by dianixn** [comment]
Paper: Luan, Dianxin, and John Thompson. "Achieving Robust Channel Estimation Neural Networks by Designed Training Data." IEEE Transactions on Cognitive Communications and Networking (2025)

github: https://github.com/dianixn/Achieving-Robust-Channel-Estimation-Neural-Networks-by-Designed-Training-Data

Paper: Luan, Dianxin, and John Thompson. "Channelformer: Attention based Neural Solution for Wireless Channel Estimation and Effective Online Training." IEEE Transactions on Wireless Communications (2023)

github: https://github.com/dianixn/Channelformer

Paper: Achieving Robust Generalization for Wireless Channel Estimation Neural Networks by Designed Training Data, ICC 2023.

github: https://github.com/dianixn/ICC_2023

Paper: Attention Based Neural Networks for Wireless Channel Estimation, VTC 2022. 

github: https://github.com/dianixn/Attention_Based_Neural_Networks_for_Wireless_Channel_Estimation

Paper: Low Complexity Channel estimation with Neural Network Solutions, WSA 2022. 

github: https://github.com/dianixn/Interpolation-ResNet

**Comment by linkpark** [comment]
您好，您的来信我已收到。

**Comment by xingtianqz** [comment]
您好，邮件已收到，谢谢。赵晓蕾

**Comment by liyiq5** [comment]
邮件已经收到，谢谢，祝好~！

**Comment by Charlie258** [comment]
你好，你的邮件已经收到，谢谢。

**Comment by WeihaoMao** [comment]
您的邮件我已经收到 感谢！

**Comment by raowei041015-glitch** [comment]
有人有基于深度学习的波束预测方面的开源代码吗，新手发现根本没有，呜呜呜

**Comment by linkpark** [comment]
您好，您的来信我已收到。

**Comment by xingtianqz** [comment]
您好，邮件已收到，谢谢.

**Comment by Charlie258** [comment]
你好，你的邮件已经收到，谢谢。

**Comment by liyiq5** [comment]
邮件已经收到，谢谢，祝好~！

**Comment by WeihaoMao** [comment]
您的邮件我已经收到 感谢！
